tests/cli/smdk_smoke_tests/smdk-basic.bats

```diff
@@ -902,4 +902,32 @@ setup_file() {
     run $SMDK_BIN test --text '444' --lookback-last '1' --record '222' --record '333' --key-value my-key
     assert_output --partial "[my-key]"
     assert_success
-}
\ No newline at end of file
+}
+
+@test "Test using `SmartModuleRecord` on fluvio-smartmodule-map-with-timestamp" {
+    # Test with smartmodule example with timestamp
+    cd "$(pwd)/smartmodule/examples/map_with_timestamp/"
+
+    # Build
+    run $SMDK_BIN build
+    refute_output --partial "could not compile"
+
+    # Test
+    run $SMDK_BIN test --text 'foo'
+    assert_output --partial "0:1970-01-01 00:00:00 UTC:foo"
```

Comment:
In this case we have to provide a way to inject the timestamp to the `SmartModuleInput` on SMDK.
As the default value for `u64` is `0` when parsing date from such value we will get `1970-01-01 00:00:00 UTC` instead.